### PR TITLE
Add Anna Balla to People

### PR DESCRIPTION
**Pull request by acballa2**
## Summary of changes
Add Anna Balla to People Page, Remove Anna Balla from Alumni Page

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Associated Issues and PRs

- Issue: #

## Associated Developers

- Dev: @

## Checklist for Reviewers

Reviewers should use [this link](/manual/guides/pull_requests) to get to the 
Review Checklist before they begin their review.

### CODE REVIEW

**Comment by acballa2** [comment]
@samgdotson wanna approve me homie?

File: _data/people.yml [hunk 1]
```diff
@@ -239,6 +239,19 @@
       icon: github-alt
       name: GitHub
 
+- name: Anna Balla
+  pic: /img/people/ballaa.png
+  dates: 2020-present
```
Comment (katyhuff) [inline _data/people.yml:244]:
```suggestion
  dates: 2019-present
```
(since you worked with us in 2019 too!)

**Review by katyhuff — CHANGES_REQUESTED** [review]

**Comment by samgdotson** [comment]
Lgtm merging

**Comment by katyhuff** [comment]
@samgdotson : In the future, please let the PR author merge review commits (and do not pull until previous reviewers have approved... ).

**Pull request merged**